packages/shopify-app-remix/src/server/authenticate/admin/billing/types.ts

```diff
@@ -155,6 +163,61 @@ export interface BillingContext<Config extends AppConfigArg> {
     options: RequireBillingOptions<Config>,
   ) => Promise<BillingCheckResponseObject>;
 
+  /**
+   * Checks if the shop has an active payment for any plan defined in the `billing` config option.
+   *
+   * @returns A promise that resolves to an object containing the active purchases for the shop.
+   *
+   * * @example
+   * <caption>Check what billing plans a merchant is subscribed to.</caption>
+   * <description>Use billing.check if you want to determine which plans are in use. Unlike `require`, `check` does not
+   * throw an error if no active billing plans are present. </description>
+   * ```ts
+   * // /app/routes/**\/*.ts
+   * import { LoaderFunctionArgs } from "@remix-run/node";
+   * import { authenticate, MONTHLY_PLAN } from "../shopify.server";
+   *
+   * export const loader = async ({ request }: LoaderFunctionArgs) => {
+   *   const { billing } = await authenticate.admin(request);
+   *   const { resp } = await billing.check({
```

Comment:
Do we return an object like `{resp: {hasActivePayment, ...}}`? That doesn't sound ideal - should we unpack that in this package and just return `{hasActivePayment, ...}`?

Reply:
No it doesn't, and I realized this when writing test but forgot to update the docs. good catch